src/test/scala/beam/agentsim/agents/choice/mode/RangeSpec.scala

```diff
@@ -1,5 +1,6 @@
 package beam.agentsim.agents.choice.mode
 import org.scalatest.FlatSpec
+import beam.sim.common.Range
```

Comment:
Is this needed? I see nothing else changed, so it makes me question this import also